daemon-tests/tests/rollover.rs

```diff
@@ -190,42 +182,110 @@ async fn maker_accepts_rollover_after_commit_finality() {
 /// The first rollover is done with `example_1`.
 /// The second rollover is done with `example_0` (we re-use it)
```

Comment:
❓ Could we retry the rollover using the `OracleData` from the first attempt? It is more natural, since when you are retrying in production you are almost certainly going to use the same announcement. It's confusing to go back in time and use the data from `example_0`.

Reply:
Initially this was on purpose because we want to make sure that the new `event-id` is used. Given that we now assert on multiple fields changing (i.e. `commit-txid`, `fees`) I think the `event-id` does not matter that much anymore and I can change it. The test still does not reflect the real world as good as it could.